### ISSUE TITLE
Portal 1 Portal Gun continues into later maps

### ISSUE THREAD

**Issue by Luke18033** [BEEmod/BEE2-items#1636]
To fix this, add logic to return to the default gun after completing the map. Unfortunately though I don't think it's possible to fix if the user manually quits or skips the map.

**Comment by Luke18033** [comment]
Update: This also applies to cube sound effects.

**Comment by Luke18033** [comment]
I don't think this is properly fixable. The best thing to do I think would be to run `sv_soundemitter_flush` at the start of every style map (rather than just P1). That would at least ensure they get switched back if you play another BEE map.

**Comment by TeamSpen210** [comment]
There is a solution actually. I can edit all of these sounds to check a sound operator variable, and based on that decide which sound set to play. So it'll still be loaded in the next map, but it'll disable itself.
